src/compojure/api/api.clj

```diff
@@ -50,14 +52,19 @@
   [& body]
   (let [[options handlers] (common/extract-parameters body false)
         options (rsc/deep-merge api-defaults options)
-        handler (apply c/routes (concat [(swagger/swagger-routes (:swagger options))] handlers))
+        raw (:raw options)
+        swagger-ui (-> (swagger/swagger-routes (:swagger options))
+                       (cond-> raw ;; if the api won't be wrapped, swagger-ui still needs to be
```

Comment:
... also, if you would do a EDN-only api for example, you could use the `swagger-routes` separately and wrap that with `wrap-json` / whatever to get JSON serialization just for that route.